UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -94,6 +92,13 @@ public static bool IsAdmin(NetworkConnection sender, out PlayerInfo player, bool
 			return true;
 		}
 
+		public static bool IsAdmin()
```

Comment:
Can you add [Client] to above this just so it will log an error if anyone uses it serverside

Reply:
But taht function is used serverside?